rca/schools/models.py

```diff
@@ -454,6 +505,26 @@ def get_programme_index_link(self):
         if programm_index:
             return f"{programm_index.get_url()}?category=related_schools_and_research_pages&value={self.pk}-{self.slug}"
 
+    def get_related_staff(self):
+        """Method to return a related staff.
+        The default behaviour should be to find related staff via the relationship
+        from StaffPage > SchoolPage. This also needs to offer the option to
+        manually add related staff to the school page, this helps solve issues
+        of custom ordering that's needed with large (>25) staff items.
+        """
+
+        from rca.people.models import StaffPage
+
+        if self.related_staff.first():
```

Comment:
It looks like `first()` is being to used to test for existence, which `exists()` is better placed for (it uses the simplest query possible). This will still generate two queries though, the general rule of thumb is that if you're going to use the results of a query anyway then don't check for existence first, so
```
related_staff = self.related_staff.select_related("image")
if related_staff:
    return related_staff
```